src/views/Settings.vue

```diff
@@ -14,18 +14,20 @@
             <ion-avatar slot="start" v-if="userProfile?.partyImageUrl">
               <Image :src="userProfile.partyImageUrl"/>
             </ion-avatar>
-            <ion-label>
-              {{ userProfile?.partyName }}
-              <p>{{ userProfile?.userLoginId }}</p>
-            </ion-label>
+            <ion-card-header>
+              <ion-card-subtitle>{{ userProfile?.userLoginId }}</ion-card-subtitle>
```

Comment:
I think we don't need to use optional chaining twice, we can follow the same way of checking as done in `ion-avatar` like applying the check only on parent, also I think `userLoginId` will always be available.

Reply:
done sir.